src/libs/actions/Report.js

```diff
@@ -1125,13 +1126,20 @@ function updateWriteCapabilityAndNavigate(report, newValue) {
  * Navigates to the 1:1 report with Concierge
  */
 function navigateToConciergeChat() {
-    // If we don't have a chat with Concierge then create it
     if (!conciergeChatReportID) {
-        navigateToAndOpenReport([CONST.EMAIL.CONCIERGE]);
-        return;
+        // In order not to delay the report life cycle, we first navigate to the unknown report
+        if (_.isEmpty(Navigation.getReportIDFromRoute())) {
+            Navigation.navigate(ROUTES.REPORT);
+        }
+        // In order to avoid creating concierge repeatedly,
+        // we need to ensure that the server data has been successfully pulled
+        Welcome.serverDataIsReadyPromise().then(() => {
+            // If we don't have a chat with Concierge then create it
+            navigateToAndOpenReport([CONST.EMAIL.CONCIERGE]);
+        });
```

Comment:
>In order to avoid creating concierge repeatedly,

What does this mean?

>we need to ensure that the server data has been successfully pulled

Server data? What server data?

Reply:
When the concierge deeplink has been opened without the report data locally after signing in as in the test steps, this code did not wait to fetch the actual reportID of the chat between user and concierge and instead it created a new optimistic chat. This however is "stale" chat then as that particular user already has existing chat with concierge with different reportID.